website/frontend/src/assets/scss/common.scss

```diff
@@ -41,33 +41,33 @@
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
-}
 
-.pagination-wrapper div:first-child {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  gap: 44px;
-}
+  div:first-child {
```

Comment:
could we use selectors by class instead of by element?

Reply:
Yes. Maybe it's will be refactor later. It's not my own css